docs/modules/engineering/pages/why_does_cerbos_use_containers.adoc

```diff
@@ -0,0 +1,26 @@
+include::ROOT:partial$attributes.adoc[]
+
+= Why do I need to run a container to use Cerbos?
```

Comment:
I feel like the justification below is more "Why do I need to run a separate process (/binary/microservice) to use Cerbos?" rather than "Why do I need to run a container"... We offer other installation methods (brew, deb, rpm, pre-built binary, compile from source) where you can run the binary directly on your machine without a Docker container.

Reply:
That's a good point, thanks.